tests/virtualization/universal/hotplugging.pm

```diff
@@ -140,8 +140,8 @@ sub test_add_vcpu {
     die "Setting vcpus failed" unless (set_vcpus($guest, 2));
     assert_script_run("ssh root\@$guest nproc | grep 2", 60);
     # Add 1 CPU
-    if ($sles_running_version eq '15' && $sles_running_sp eq '3' && is_xen_host && is_fv_guest($guest)) {
-        record_soft_failure('bsc#1180350 Failed to set live vcpu count on fv guest on 15-SP3 Xen host');
```

Comment:
Has the old bug bsc#1180350 been verified on 15sp3? We still have 15sp3 host tests in 15sp4 LTP. If still valid bug, we need to keep the soft fail for it and add new one for 15sp4. Otherwise, current change should be taken.

Reply:
Good point,  confirm bsc#1180350 does still reproduce on sles15sp3 xen server with sles15sp4 fv guest. updated bsc#1180350 as REOPENED and added the comments now. 
Also, updated code to included bsc#1180350  -> sles15sp3 and bsc#1188898 -> sles15sp4 together on OSD. 
Add the latest verification run as below:
[gi-guest_sles12sp5-on-host_developing-xen](https://openqa.suse.de/tests/6655365#)
[gi-guest_sles15sp3-on-host_developing-xen](https://openqa.suse.de/tests/6655366#)
[gi-guest_developing-on-host_developing-xen](https://openqa.suse.de/tests/6655367#)
[gi-guest_developing-on-host_sles15sp3-xen](https://openqa.suse.de/tests/6655368#)